### PR TITLE
fix: buttons on list card

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
Revert to previous designs

### CODE REVIEW

File: packages/app/components/claim/claim-button.tsx [hunk 1]
```diff
@@ -320,7 +319,7 @@ export const ClaimButton = ({
             tw="ml-1 text-sm font-semibold leading-5"
             style={{ color: isDark ? colors.black : colors.white }}
           >
-            {isAuthenticated ? "Pre-Save to Collect" : "Pre-Save on Spotify"}
+            Pre-Save on Spotify
```
Comment (hirbod) [inline packages/app/components/claim/claim-button.tsx]: What about this line?

**Review by hirbod — COMMENTED** [review]
Did you test this also on small devices? iPhone SE SIM maybe?

File: packages/app/components/claim/claim-button.tsx (hunk 1 above)
Comment (intergalacticspacehighway) [reply packages/app/components/claim/claim-button.tsx]: yeah thought to make the copy a bit consistent. Pre-Save to collect actually pre-saves on Spotify so thought we should maybe change it. wdyt?

**Comment by intergalacticspacehighway** [comment]
> Did you test this also on small devices? iPhone SE SIM maybe?

Nope, it's breaking there pretty bad!

**Comment by hirbod** [comment]
Do you have a screenshot?

**Review by hirbod — APPROVED** [review]
LGTM

**Pull request merged**